.vscode/scripts/run-mosquitto.sh

```diff
@@ -14,18 +14,20 @@
 # SPDX-License-Identifier: Apache-2.0
 
 echo "#######################################################"
-echo "### Running Mosquitto                               ###"
+echo "### Running MQTT Broker                             ###"
 echo "#######################################################"
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-#Terminate existing running VAL services
-RUNNING_CONTAINER=$(docker ps | grep "eclipse-mosquitto" | awk '{ print $1 }')
-MOSQUITTO_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.version | tr -d '"')
+MOSQUITTO_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.image | tr -d '"')
+MOSQUITTO_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .mosquitto.version | tr -d '"')
 
+#Terminate existing running services
+RUNNING_CONTAINER=$(docker ps | grep "$MOSQUITTO_IMAGE" | awk '{ print $1 }')
```

Comment:
Please add Help() function to explain the parameters

Maybe we need to add verification in front to make sure that parameter is provided and in case of failure - clear error message shall be shown

Reply:
I woudn't like to spend the effort, as we will change this (hopefully) soon with the migration to lifecycle management.
Furthermore, those scripts are not to be called directly, but via VSCode tasks only